charts/uptrace/values.yaml

```diff
@@ -16,6 +16,10 @@ clickhouse:
     repository: clickhouse/clickhouse-server
     pullPolicy: IfNotPresent
     tag: '22.10'
+  persistence:
+    enabled: true
```

Comment:
Will this break existing installations? Does it make sense to disable this by default to avoid it?

Reply:
Note that for example bitnami charts all seems to default to enabled persistence with the default storage class like we do here.

Reply:
Is there a reason to not just the bitnami chart instead? Less maintenance for such a dependency.

Reply:
@xvilo no reasons, I've just took the official Docker repo and is that's what we have. Feel free to send a PR.